package.json

```diff
@@ -9,6 +9,7 @@
     "atqr:start:api": "nx serve api",
     "postatqr:start:api": "docker compose down",
     "atqr:start:web": "yarn parcel apps/atqr/web/src/pages/**/*.pug --open",
+    "atqr:start:test": "yarn run jest",
```

Comment:
"start" is generally used to start the application. Rename the task to simply "atqr:test"